### PR TITLE
Yearn Dune Bounty Part 1

### PR DESCRIPTION
**Pull request by 3StepCapital**
I've checked that:

* [ ] the query produces the intended results
* [ ] the folder name matches the schema name
* [ ] the schema name exists in Dune
* [ ] views are prefixed with `view_`, functions with `fn_`.
* [ ] the filename matches the defined view, table or function and ends with .sql
* [ ] each file has only one view, table or function defined  
* [ ] column names are `lowercase_snake_cased`

* Added new tokens to tokens.sql
* Created a new view with many yearn vaults for others to access in Dune

### CODE REVIEW

File: ethereum/yearn/all_vaults.sql [hunk 1]
```diff
@@ -0,0 +1,154 @@
+BEGIN;
+
+CREATE TABLE IF NOT EXISTS yearn."yearn_all_vaults" (
```
Comment (chuxinh) [inline ethereum/yearn/all_vaults.sql:3]: just following styles in other dune tables
```suggestion
CREATE TABLE IF NOT EXISTS yearn.yearn_all_vaults (
```

File: ethereum/yearn/all_vaults.sql [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+    contract_address	bytea UNIQUE,
+    ytoken text,
+    decimals numeric,  
+    symbol	text, 
+    tag text
+);
+
+--DELETE FROM dune_user_generated."yearn_all_vaults" *;
```
Comment (chuxinh) [inline ethereum/yearn/all_vaults.sql:11]: delete from this table?

File: ethereum/yearn/all_vaults.sql [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+
+
+COPY yearn."yearn_all_vaults"(contract_address, ytoken, decimals, symbol, tag) FROM stdin;
```
Comment (chuxinh) [inline ethereum/yearn/all_vaults.sql:14]:
```suggestion
COPY yearn.yearn_all_vaults(contract_address, ytoken, decimals, symbol, tag) FROM stdin;
```

**Review by chuxinh — APPROVED** [review]
Vaults looking good to me with a few nits.
Still need to tag the Dune team to review before merging part I. Thanks!

**Comment by masquot** [comment]
@3StepCapital This PR contains previous commits not related to Yearn. 

Thank you for creating a new PR starting off from the current abstractions `master` branch with only commits and file changes related to Yearn

**Pull request closed without merging**